db/migrate/20221003172937_add_email_to_reservations.rb

```diff
@@ -0,0 +1,12 @@
+class AddEmailToReservations < ActiveRecord::Migration[7.0]
+  class Reservation < ActiveRecord::Base
+    end
+
+    def change
+      add_column :reservations, :email, :string
+      Reservation.reset_column_information
+      Reservation.all.each do |reservation|
+        reservation.update(:email => 'example@example.com'  )
+      end
+    end
```

Comment:
I do not think updating every existing reservation to some sort of example email is a good idea in a migration, it would never pass in a real application, 
If you assume there is no production yet an you have no production reservations to maintain then I would simply clear local database and seed it again or just remove all reservations you have through rails console

Reply:
This part was based on rails guides migrations, so even more thanks to you on pointing out how and why that may not be the best approach